### ISSUE TITLE
Table containing the verification rows does not have a caption.

### ISSUE THREAD

**Issue by pushyamig** [tl-its-umich-edu/canvas-course-manager-next#257]
Category: Semantics and Structure. Impact: major

Location: Confirmation screen: Gradebook formatter, Add UM user, Create sections

Issue
Table containing the verification rows does not have a caption.

Impact
Given the SPA nature of CCM, lack of caption will add cognitive burden. What happened? 

Solution
Provide a caption. Indicating the number of items returned would be valuable. Caption can be visually hidden if needed for design considerations

**Comment by ssciolla** [comment]
@janelilr and @pushyamig , I am working on adding `caption` elements to tables. Would you like these hidden (screenreader only) or visible? This is the default styling:
<img width="855" alt="Screen Shot 2022-01-06 at 12 11 32 PM" src="https://user-images.githubusercontent.com/35741256/153259679-4950a635-364d-42e5-b5fb-c0a4ad02debe.png">
 
I was thinking -- as the issue suggests -- just doing something like "Data was found for X enrollments/sections/gradebook records", or "X validation or API errors occurred."

**Comment by pushyamig** [comment]
>Would you like these hidden (screenreader only) or visible?
only for screenreader.

**Comment by gsilver** [comment]
Nice captions!
